godel_robots/abb/godel_irb2400/godel_irb2400_moveit_config/config/sensors_rgbd.yaml

```diff
@@ -0,0 +1,31 @@
+sensors:
+  - sensor_plugin: occupancy_map_monitor/PointCloudOctomapUpdater
+    # TODO: use appropriate cloud input topic
+    point_cloud_topic: godel_octomap_input
```

Comment:
This topic name is a place-holder: it can (obviously) be changed to something else.